buildshiprun/handler.go

```diff
@@ -94,7 +94,14 @@ func Handle(req []byte) string {
 		sdk.PostAudit(auditEvent)
 
 		status.AddStatus(sdk.StatusFailure, err.Error(), sdk.BuildFunctionContext(event.Service))
-		reportStatus(status)
+		if event.SCM == "github" {
+			reportStatus(status)
```

Comment:
Should better return error from both methods and log here, or log within the methods on both places.

Why not move the SCM check within a single reportStatus method?

Reply:
This is not function I wrote and it was like that before so I left it

Reply:
I will change the name `reportStatus` to `reportGitHubStatus` and move the logs of `reporGitLabStatus` inside the log of the function body. Is that ok @ivanayov